backend/models/taskSchema.js

```diff
@@ -23,4 +34,4 @@ const TaskSchema = new mongoose.Schema(
 );
 
 const Task = mongoose.model("Task", TaskSchema);
-module.exports = Task;
```

Comment:
We want to export Task, not Task Schema here

Reply:
Tests require the actual schema being exported, which is why a variable was added to taskservicees